packages/rocketchat-theme/client/imports/forms/input.css

```diff
@@ -37,6 +37,10 @@
 			padding: 0.75rem 1rem 0.75rem 2.75rem;
 		}
 
+		& + .rc-input__textarea {
```

Comment:
Is it necessary to change the Rocket.Chat-CSS? I expect some merge conflicts here in future.
Shall we use an own selector in our custom CSS or is this something related to all text areas

Reply:
@mrsimpson We could use our own selector. However, was bit reluctant about duplicating the similar group of properties. Let me check if we can somehow reference to this properties from a our own css.

Reply:
@mrsimpson Locally moved to css selector which will avoid merge conflicts.